Kitodo/src/main/java/org/kitodo/production/services/data/ProjectService.java

```diff
@@ -301,6 +301,21 @@ public String getProjectTitles(List<Project> projects) throws DAOException {
         }
     }
 
+    /**
+     * Checks whether the given project has any processes assigned to it.
+     *
+     * @param projectId
+     *          the ID of the project to check
+     * @return return true if at least one process belongs to the project,
+     * false otherwise
+     */
+    public boolean hasProcesses(int projectId) throws DAOException{
```

Comment:
- Missing white space between `DAOException` and `{` character.
- Having tests for this new method should be made as long an test for zero and an other test for at least one process assigned to a project. A test for the throwing of the `DAOException` would be difficult and can be omitted but if you found a simple way you can add this test too.